packages/fxa-auth-server/scripts/convert-customers-to-stripe-automatic-tax/convert-customers-to-stripe-automatic-tax.ts

```diff
@@ -265,4 +312,17 @@ export class StripeAutomaticTaxConverter {
       report.nextInvoice,
     ];
   }
+
+  /**
+   * Appends the report to the output file
+   * @param report an array representing the report CSV
+   */
+  async writeReport(report: (string | number | null)[]) {
```

Comment:
Is it possible to add a header to the CSV?

Reply:
Could add that, yep. Is that something we desire?

Reply:
Oh this isn't super important.

I guess it depends what the report is used for. If it's for human consumption, it makes it a little easier to consume if there's a heading for the column.

Reply:
Pushed an update that includes a header

Reply:
Ended up removing the header since it means that the script couldn't be interrupted and re-run.